iothub/device/src/ClientFactory.cs

```diff
@@ -47,7 +47,7 @@ public static InternalClient Create(string hostname, string gatewayHostname, IAu
         /// </summary>
         /// <param name="hostname">The fully-qualified DNS hostname of IoT Hub</param>
         /// <param name="authenticationMethod">The authentication method that is used</param>
-        /// <param name="transportType">The transportType used (Http1 or Amqp)</param>
+        /// <param name="transportType">The transportType used (Http1, Amqp or Mqtt)</param>
```

Comment:
Could we use a url to point to the definition of transport type? I believe there are more right? https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/TransportType.cs

Reply:
Amqp and Mqtt cover both the Tcp and WS variants. By defining the transport type as Amqp, the device client will create a prioritized transport list of Amqp_tcp, Amqp_ws (same for Mqtt).

Reply:
I can add a link to this file, sure